packages/stylus/BUILD.bazel

```diff
@@ -0,0 +1 @@
+# Marker that this directory is a bazel package
```

Comment:
`npm_package` rule and also add to `scripts/packages.sh`? At one point I tried adding the release script to CI but bazel was crashing in CircleCI and haven't track it down yet.